src/import/trace-event.test.ts

```diff
@@ -122,3 +122,7 @@ test('importTraceEvents invalid x nesting', async () => {
 test('importTraceEvents event reordering name match', async () => {
   await checkProfileSnapshot('./sample/profiles/trace-event/event-reordering-name-match.json')
 })
+
+test('importTraceEvents simple json profile', async () => {
```

Comment:
I can add a test for supporting multiple threads but I don't know where to get it - @jlfwong do you know where to get one?

Reply:
Added test [here](a565f1d4eb73ac1e92e4cf8275da4bf48dda8c26)